app/lib/widgets/cards/productCard/shared/product_notation.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flex_ui/flex_ui.dart';
+import 'package:flutter/material.dart';
+
+class FlexProductNotation extends StatelessWidget {
```

Comment:
Can we name `FlexProductRating` ? p.s. love the Semantics usage

Reply:
Done !